src/applications/personalization/profile/mocks/server.js

```diff
@@ -236,7 +236,8 @@ const responses = {
     //   .json(serviceHistory.generateServiceHistoryError('403'));
   },
   'GET /v0/disability_compensation_form/rating_info': (_req, res) => {
-    return res.status(200).json(ratingInfo.success);
+    // return res.status(200).json(ratingInfo.success.serviceConnected0);
+    return res.status(200).json(ratingInfo.success.serviceConnected40);
     // return res.status(500).json(genericErrors.error500);
```

Comment:
Fixes an issue where the rating info was correctly mocked but not correctly returned